ape_alchemy/providers.py

```diff
@@ -12,11 +12,12 @@
 
 from .exceptions import AlchemyFeatureNotAvailable, AlchemyProviderError, MissingProjectKeyError
 
-_ETH_ENVIRONMENT_VARIABLE_NAMES = ("WEB3_ALCHEMY_PROJECT_ID", "WEB3_ALCHEMY_API_KEY")
-_ARB_ENVIRONMENT_VARIABLE_NAMES = (
-    "WEB3_ARBITRUM_ALCHEMY_PROJECT_ID",
-    "WEB3_ARBITRUM_ALCHEMY_API_KEY",
-)
+_ENVIRONMENT_VARIABLE_OPTIONS = {
+    "ethereum": ("WEB3_ALCHEMY_PROJECT_ID", "WEB3_ALCHEMY_API_KEY"),
```

Comment:
So I am just now realizing that Alchemy has differnent API keys per network whereas Infura does not.
Alchemy has different keys even bettwen `ethereum:goerli` and `ethereum:mainnet`. Should we have support per network keys rather than per ecosystem here then?

Reply:
Yeah probably, ugh

Reply:
No problem, it is done